docs/admin/admin.yaml

```diff
@@ -72,8 +257,8 @@ paths:
                 $ref: '#/components/Error'
               example:
                 status: 400
-                code: 'INVALID_ID'
-                message: 'ID is invalid.'
+                code: INVALID_ID
```

Comment:
if you decided to remove here quotes, then they should be removed everywhere, gotta stick to the same pattern everywhere
u - uniformity